### DISCUSSION TITLE
Applying JSON Schema Validation Consistently Across Form and API Inputs Using @rjsf/validator-ajv8

### DISCUSSION THREAD

**Discussion by snicol21** [rjsf-team/react-jsonschema-form#4186] [General]
I am currently utilizing `react-jsonschema-form` for validating data that users submit through a form, which is then sent to my database via an API. While the form is the primary method for data entry, there are potential use cases where data might be sent directly to our API, bypassing the form. I would like to ensure that the same JSON validation applies to both data entry methods. Is it possible to make the [@rjsf/validator-ajv8](https://www.npmjs.com/package/@rjsf/validator-ajv8) available as a function that I could use to validate API requests directly? This would help maintain consistent data validation across different input methods.

**Comment by nickgros** [comment]
Does `@rjsf/validator-ajv8` not work on your server? It shouldn't be using any browser-specific APIs, so you should be able to create a `validator` instance using the package and run `validateFormData()`.
